csrc/cutlass_extensions/common.hpp

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "cutlass/cutlass.h"
+#include <climits>
+#include "cuda_runtime.h"
+#include <iostream>
+
+/**
+ * Helper function for checking CUTLASS errors
+ */
+#define CUTLASS_CHECK(status)                        \
+  {                                                  \
```

Comment:
Maybe extract status first (like below) so this macro can directly wrap expressions like function calls and not double-evaluate them?

Reply:
Done